### PR TITLE
Fix Tython CLI not showing results of validation

### PR DESCRIPTION
**Pull request by earafat-oak9**
v1.0.12 of the python SDK broke the runner output, which meant that the CLI wasn't able to show the results of the validation anymore.

After the runner output was fixed, the CLI still wasn't able to show the results correctly, because the runner output now is using the proto-based Finding object, so because of that interface change, the CLI had to be updated as well.

### CODE REVIEW

File: py/core/utilities.py [hunk 1]
```diff
@@ -38,10 +39,11 @@ def persist_runner_output(args_path: str, runner_stdout: StringIO, blueprint_pro
...
 
-    # if findings:
-    #     for finding in findings:
-    #         if finding:
-    #             findings_json_list.append(finding.__json__())
+    if findings:
+        for finding in findings:
+            if finding:
+                findings_json_list.append(MessageToDict(finding))
+
```
Comment (earafat-oak9) [inline py/core/utilities.py:46]: I had this commented out in v1.0.12 thinking it was just for debugging purposes...

**Review by bnicoll-oak9 — APPROVED** [review]

**Review by KirkRehalOak9 — APPROVED** [review]

**Review by KirkRehalOak9 — APPROVED** [review]

File: cli/internal/viewers/findings_viewer.go [hunk 2]
```diff
@@ -217,7 +219,7 @@ func (viwer FindingsViewer) groupByResourceName(findingList []runner.Finding) (g
 }
 
 func countSeverity(finding runner.Finding, counts map[string]int) {
-	switch finding.Rating {
+	switch finding.Severity {
```
[the remark this answers is not in the document]
Comment (earafat-oak9) [reply cli/internal/viewers/findings_viewer.go:222]: If the point of contention was UX, then that should be solved by an abstraction, but the data model should be clear about what it accepts.

**Comment by earafat-oak9** [comment]
Are there tests that should run, or do I merge and hope for the best?

File: cli/internal/viewers/findings_viewer.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (bnicoll-oak9) [reply cli/internal/viewers/findings_viewer.go:222]: We're on the same page about this, I separated the two with the intention that we would sweep our dirty data under the rug. The UX concerns should be solved with the strongly-typed "createFinding()" and "createKudos()"

**Pull request merged**